### PR TITLE
Feature 1.0/checkbox as toggle

### PR DESCRIPTION
**Pull request by kunal-adobe**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--{repo}--{owner}.hlx.live/
- After: https://<branch>--{repo}--{owner}.hlx.live/

### CODE REVIEW

File: blocks/form/functions.js [hunk 1]
```diff
@@ -7,16 +7,71 @@
  */
 function getFullName(firstname, lastname) {
   // eslint-disable-next-line no-param-reassign
-  firstname = (firstname == null) ? '' : firstname;
+  firstname = firstname == null ? "" : firstname;
```
Comment (srivastava21ashish) [inline blocks/form/functions.js:10]: use single quotes

File: blocks/form/form.css [hunk 2]
```diff
@@ -1,271 +1,273 @@
 :root {
```
Comment (srivastava21ashish) [inline blocks/form/form.css:1]: Write your CSS in styles.css and import it in forms.css. No changes directly in forms.css please. 
Also I see a lot of formatting isues.

File: blocks/form/functions.js [hunk 3]
```diff
@@ -7,16 +7,71 @@
  */
```
Comment (srivastava21ashish) [inline blocks/form/functions.js:7]: Execute npm run lint and fix all liniting issues, and then raise your PR

Comment (srivastava21ashish) [reply blocks/form/functions.js:7]: Lint issues in file:
/home/runner/work/hdfc-bank/hdfc-bank/blocks/form/functions.js
Error:   10:35  error    Strings must use singlequote  quotes
Error:   12:33  error    Strings must use singlequote  quotes
Error:   13:27  error    Strings must use singlequote  quotes
Error:   27:48  error    Strings must use singlequote  quotes
Error:   32:39  error    Strings must use singlequote  quotes
Warning:   39:5   warning  Unexpected console statement  no-console
Error:   46:50  error    Strings must use singlequote  quotes
Error:   47:32  error    Strings must use singlequote  quotes
Error:   47:56  error    Strings must use singlequote  quotes
Warning:   49:5   warning  Unexpected console statement  no-console
Error:   50:86  error    Missing trailing comma        comma-dangle
Error:   59:47  error    Strings must use singlequote  quotes
Error:   71:5   error    Strings must use singlequote  quotes
Error:   72:5   error    Strings must use singlequote  quotes
Error:   72:38  error    Missing trailing comma        comma-dangle

**Review by srivastava21ashish — CHANGES_REQUESTED** [review]

**Review by mainiaditaya — CHANGES_REQUESTED** [review]
add URL for testing

https://develop--hdfc-bank--aemsites.hlx.page/content/forms/af/hdfc
2. discussed the fix for the async function call. kindly add the same.

**Comment by mainiaditaya** [comment]
reverted for the async function fix.

**Pull request closed without merging**